typescript/sdk/src/ica/schemas.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from 'zod';
+
+import { ZHash } from '../metadata/customZodTypes.js';
+import { DerivedOwnableSchema } from '../schemas.js';
+
+export const RemoteIcaRouterConfigSchema = z.record(
```

Comment:
Is this the same as https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/455a897fb50f88812ca3842ac0adb41cce817eb4/typescript/sdk/src/router/types.ts#L80-L83

Can we reuse?

Reply:
solved in dms